queries.sql

```diff
@@ -0,0 +1,44 @@
+--Find all animals whose name ends in "mon".
+
+select * from animals where name like '%mon';
+--   1 | Agumon  | 2020-02-03    |               0 | t        |     10.23
+--   2 | Gabumon | 2018-11-15    |               2 | t        |       8.0
+--   4 | Devimon | 2017-05-12    |               5 | t        |      11.0
+
+--List the name of all animals born between 2016 and 2019.
+
+select name from animals where date_of_birth between '2016-01-01'and '2019-12-31';
+--  Gabumon
+--  Devimon
+
+--List the name of all animals that are neutered and have less than 3 escape attempts.
+select name from animals where neutered= true and escape_attempts<=3;
```

Comment:
- [ ] Just as the question states, you should list all animals that have less than 3 escape attempts. Your query is currently, listing animals who have less than 3 and those who have exactly 3 escaped attempts. And that shouldn't be the case.
Please remove the `=` symbol from your query.